HopsanGenerator/templates/fmu1CsImportComponentTemplate.hpp

```diff
@@ -180,6 +180,14 @@ class <<<className>>> : public <<<classParent>>>
             return;
         }
 
+        //Set parameters
```

Comment:
This looks like an important bugfix, is it something that needs special mentioning in the release notes ?

Reply:
We should mention that parameters are now supported for FMI import, both version 1 and 2. Or did you think of something else?

Reply:
Yes that is what I meant, I will add it to the release notes